image/verify/src/verifier.rs

```diff
@@ -749,7 +750,7 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
 
         let log_info: ImageSvnLogInfo = ImageSvnLogInfo {
             manifest_svn: verify_info.svn,
-            manifest_min_svn: verify_info.min_svn,
+            manifest_deprecated_min_svn: verify_info.deprecated_min_svn,
```

Comment:
We should just rename the field to 'reserved' in the image and remove it from verifier function, test cases etc.